### ISSUE TITLE
Importing custom python package and miniconda installation prompt issue

### ISSUE THREAD

**Issue by sarathsund** [rstudio/reticulate#1213]
I able to import a custom python package internally created. 
while doing so, (highlighted in bold)
1. There is a miniconda installation prompt and I suppose it looks to download the python package - how to suppress this ? 

2.  Also it looks for deactivate-r-base.sh & activate-r-base.sh file - is there any configurations changes to be done ?

R version 4.1.3 (2022-03-10) -- "One Push-Up"
Copyright (C) 2022 The R Foundation for Statistical Computing
Platform: x86_64-conda-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> library(reticulate)
**news <- import("lexisnexis.dataset.news")**
dataset <- news$Dataset()
request <- news$Request()
response <- dataset$get(request)$data
typeof(response)
No non-system installation of Python could be found.
Would you like to download and install Miniconda?
Miniconda is an open source environment management system for Python.
See https://docs.conda.io/en/latest/miniconda.html for more details.

Would you like to install Miniconda? [Y/n]: n
Installation aborted.
**/tmp/RtmpDw5bbj/file2973fa067f4.sh: 4: /opt/conda/etc/conda/deactivate.d/deactivate-r-base.sh: [[: not found
/tmp/RtmpDw5bbj/file2973fa067f4.sh: 5: /opt/conda/etc/conda/activate.d/activate-r-base.sh: [[: not found
/tmp/RtmpDw5bbj/file2973fa067f4.sh: 4: /opt/conda/etc/conda/deactivate.d/deactivate-r-base.sh: [[: not found
/tmp/RtmpDw5bbj/file2973fa067f4.sh: 5: /opt/conda/etc/conda/activate.d/activate-r-base.sh: [[: not found
/tmp/RtmpDw5bbj/file2975a579e38.sh: 4: /opt/conda/etc/conda/deactivate.d/deactivate-r-base.sh: [[: not found
/tmp/RtmpDw5bbj/file2975a579e38.sh: 5: /opt/conda/etc/conda/activate.d/activate-r-base.sh: [[: not found
/tmp/RtmpDw5bbj/file2975a579e38.sh: 4: /opt/conda/etc/conda/deactivate.d/deactivate-r-base.sh: [[: not found
/tmp/RtmpDw5bbj/file2975a579e38.sh: 5: /opt/conda/etc/conda/activate.d/activate-r-base.sh: [[: not found**
2022-05-17 17:43:51 - INFO - TransactionId = ce7aaa36-a8a5-426f-9186-5cd9f9b8b554
Error in py_call_impl(callable, dots$args, dots$keywords) : 
  lexisnexis.exception.exceptions.WorkspaceConnectorException: ConnectionError(MaxRetryError("HTTPConnectionPool(host='172.28.25.126', port=8080): Max retries exceeded with url: /1.0/dataset/info (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd81d1e07f0>: Failed to establish a new connection: [Errno 111] Connection refused'))"))
> library(reticulate)
news <- import("lexisnexis.dataset.news")
dataset <- news$Dataset()
request <- news$Request()
response <- dataset$get(request)$data
typeof(response)
2022-05-17 17:44:52 - INFO - TransactionId = 9fc0f380-c8da-4518-9ca3-d47092a4e63b
2022-05-17 17:45:12 - INFO - Fetching articles from workspace, id=7957643e-59be-49f1-af1f-d60581a7dfef
2022-05-17 17:45:12 - INFO - Caching 9999 articles.
2022-05-17 17:45:12 - INFO - Loading.. 20%

> reticulate::py_config()
python:         /opt/conda/bin/python3
libpython:      /opt/conda/lib/libpython3.9.so
pythonhome:     /opt/conda:/opt/conda
version:        3.9.12 | packaged by conda-forge | (main, Mar 24 2022, 23:25:59)  [GCC 10.3.0]
numpy:          /opt/conda/lib/python3.9/site-packages/numpy
numpy_version:  1.22.3
lexisnexis:     /opt/conda/lib/python3.9/site-packages/lexisnexis

python versions found: 
 /opt/conda/bin/python3
 /opt/conda/bin/python

> utils::sessionInfo()
R version 4.1.3 (2022-03-10)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: Ubuntu 20.04.4 LTS

Matrix products: default
BLAS/LAPACK: /opt/conda/lib/libopenblasp-r0.3.20.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] reticulate_1.25

loaded via a namespace (and not attached):
 [1] compiler_4.1.3  here_1.0.1      Matrix_1.4-1    rprojroot_2.0.3
 [5] rappdirs_0.3.3  Rcpp_1.0.8.3    grid_4.1.3      jsonlite_1.8.0 
 [9] png_0.1-7       lattice_0.20-45

**Comment by t-kalinowski** [comment]
Hi! Thank you for filing.

To disable the installation prompt, you can set the environment variable `RETICULATE_MINICONDA_ENABLED=0`. You can also force reticulate reticulate to use a particular python binary by setting the environment variable `RETICULATE_PYTHON=/path/to/bin/python` (e.g., in .profile, .bashrc, .bash_profile, .Renviron, .Rprofile, or any one of 20 other such places).

The error about activate-r-base.sh not being found is, I believe, originating from a `conda activate` command--reticulate itself doesn't look for activate-r-base.sh.

We're not very heavy users of conda, so PR's are welcome as you track this down :)

**Comment by miraculixx** [comment]
Unfortunately setting neither, `RETICULATE_MINICONDA_ENABLED=0` nor `RETICULATE_PYTHON` works:

```
(base) jovyan@cb6b9e55980b:/app$ Rscript test.R
/tmp/Rtmp4fUL7C/file864315c63ce.sh: 4: /opt/conda/etc/conda/deactivate.d/deactivate-r-base.sh: [[: not found
/tmp/Rtmp4fUL7C/file864315c63ce.sh: 5: /opt/conda/etc/conda/deactivate.d/deactivate-gxx_linux-64.sh: Syntax error: "(" unexpected
Error in Sys.setenv(PATH = new_path) : wrong length for argument
Calls: import ... py_discover_config -> python_config -> python_munge_path -> Sys.setenv
In addition: Warning message:
In system2(Sys.which("sh"), fi, stdout = if (identical(intern, FALSE)) "" else intern) :
  running command ''/usr/bin/sh' /tmp/Rtmp4fUL7C/file864315c63ce.sh' had status 2
Execution halted
```

```
# test.R
library(reticulate)
pysys <- import("sys")
```

This is running the [jupyterstacks/base-notebook](https://hub.docker.com/r/jupyter/base-notebook/) with the R-essentials package installed in addition.

**Comment by t-kalinowski** [comment]
I believe this was fixed with #1255.
